src/test-helpers/queries.ts

```diff
@@ -0,0 +1,17 @@
+import { findAllByText, screen } from '@testing-library/dom'
+
+/**
+ * Gets a thought by value.
+ */
+export const findThoughtByText = async (value: string, container?: HTMLElement) => {
+  const thoughtNodes = await (container ? findAllByText(container!, value, { exact: true }) : screen.findAllByText(value, { exact: true }))
+  return thoughtNodes.find(t => t.hasAttribute('contenteditable'))
+}
+
+/**
+ * Gets a thought by value.
```

Comment:
Make sure comments are in alignment with the code, as it can represent a point of confusion for other developers if it doesn't make sense.